README.md

```diff
@@ -36,11 +36,6 @@ If you’re using this demo, please **★Star** this repository to show your int
 ```
 PROJECT_ID="<your-project-id>"
 gcloud services enable container.googleapis.com --project ${PROJECT_ID}
-gcloud services enable monitoring.googleapis.com \
-    cloudtrace.googleapis.com \
-    clouddebugger.googleapis.com \
-    cloudprofiler.googleapis.com \
-    --project ${PROJECT_ID}
```

Comment:
Praise: Good catch. I didn't realize this (enablement of GCP operations services) was still here in the quickstart.